octoprint_mrbeam/static/js/mother_viewmodel.js

```diff
@@ -230,6 +234,8 @@ $(function () {
                  self.terminal.checkAutoscroll();
             });
             self.terminal.activeAllFilters();
+
+            self.state._processProgressData = function(){};
```

Comment:
```suggestion
            // MR_BEAM_OCTOPRINT_PRIVATE_API_ACCESS
            // our implementation here should be used instead of octoprints
            // to fix issues with the laser job time display
            self.state._processProgressData = function(){};
```